### DISCUSSION TITLE
太棒了 怎么捐赠支持

### DISCUSSION THREAD

**Discussion by mpoes** [KIRAKIRA-DOUGA/KIRAKIRA-Cerasus#238] [Show and tell]
有群吗？

**Comment by Aira-Sakuranomiya** [comment]
目前聊天群暂不开放，等到我们把网站后台的部分功能做出来了才行（跟入群规则有关）

捐赠我们需要考虑一下使用什么渠道比较合适

**Comment by mpoes** [comment]
OK 等你们好消息

**Comment by Aira-Sakuranomiya** [comment]
现在聊天群已经开放了：977172105

在 https://kirakira.moe/settings/dashboard 查询信息并按照验证格式填入验证答案即可
